Not working on Modulus with Meteor 1.2.x

After update to 1.2, must remove this package for app to work.
Do you have a sense of when you might be able to resolve this?  This is mission-critical for us.

Thank you for your help, and your helpful package.

When I run the code locally, I don't get any errors, but the functionality to pull from a spreadsheet doesn't work.
When I deploy to Modulus, I get the errors:
npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/1to2
npm ERR! 404 
npm ERR! 404 '1to2' is not in the npm registry.

I searched for 1to2 and couldn't find anything about it.
I ran npm locally and did a build locally.  This allowed me to find the 1to2.js file, which upon build ends up within the meteor-google-spreadsheets package.
It's buried so many folders deep that it'd be hard to indicate where.  If you need me to, I can give you the full path, but I'd have to hand type a 20-deep subdirectory... I presume that if you search you can find it on your own machine more easily.

Thanks for your help with this.

Thanks for your help.  Unfortunately, on deploy to Modulus, I'm still getting the error:
npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/1to2
npm ERR! 404 
npm ERR! 404 '1to2' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'BrainfulApp'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

--- a/server/methods.js
+++ b/server/methods.js
@@ -45,7 +45,7 @@ Meteor.methods({
   },
   'spreadsheet/fetch2': function (spreadsheetName, worksheetId, options) {
     check(spreadsheetName, String);
-    check(worksheetId, String);
+    check(worksheetId, Match.OneOf(String, Number));
     check(options, Object);
     var fut = new Future(); //don't return until we're done exporting
 
@@ -76,7 +76,7 @@ Meteor.methods({
   },
   'spreadsheet/update': function (spreadsheetName, worksheetId, updateObject, options) {
     check(spreadsheetName, String);
-    check(worksheetId, String);
+    check(worksheetId, Match.OneOf(String, Number));
     check(updateObject, Object);
     check(options, Object);
     var fut = new Future(); //don't return until we're done exporting

